.all-contributorsrc

```diff
@@ -694,6 +698,12 @@
         "userTesting"
       ]
     },
+    {
```

Comment:
I'm not sure what's going on here, but Christine is now added in multiple times.

Reply:
Hi Bekah I believe I resolved the requested change to the .all-contributorsrc file. Let me know if there are anymore changes you need from me. Thank you!